README.md

```diff
@@ -23,12 +23,15 @@ Our aim is to provide a single source for the best possible OS X binaries for cg
 * autoconf
 * libtool
 * GNU sed
+* libevent
```

Comment:
This package should not be required to build anything. What prompted adding this to the prereqs?

Reply:
This looks like an issue similar to libusb - this isn't a rereq, it's just not being picked up in the `output/` directory but the `configure` script for BFGMiner.

Reply:
I got the error:
```
configure: error: Package requirements (libevent >= 2.0.3) were not met:

No package 'libevent' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables libevent_CFLAGS
and libevent_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
```
Maybe it just needs linking

Reply:
Please remove the libevent additions to the readme and instead update the `build_miner.sh` to pass these after the existing `CFLAGS` args:

```
    libevent_CFLAGS="-I$BUILD_DIR/output/include/event2" \
    libevent_LIBS="-L$BUILD_DIR/output/lib -levent" \
```

Reply:
@fabulouspanda see my follow-up above please

Reply:
That didn't work for me, but adding 
```
PKG_CONFIG_PATH=$BUILD_DIR/output/lib/pkgconfig
```
got it linked up